### ISSUE TITLE
Enterprise Card Input field malfunction

### ISSUE THREAD

**Issue by kiranBehara1809** [fireship-io/fireship.io#1605]
Hey Buddy, I have found an issue in your enterprise card .
There is a input type number box and "+" and "-" buttons on either side of this input field.
If I increment by using the native increment control provided by HTML and then click "+" button, the value directly changes to "50"
Thanks in advance.
<img width="444" alt="image" src="https://github.com/fireship-io/fireship.io/assets/68887756/a571d1da-ea33-460a-8965-2860e25ca2d0">

**Comment by Sahadev18** [comment]
Yes, when the value is manually entered and incremented using the + button it automatically reaches the max limit that is 50.
